holidays/locale/th/LC_MESSAGES/BN.po

```diff
@@ -78,3 +80,8 @@ msgstr "วันขึ้นปีใหม่อิสลาม"
 #. Birth of the Prophet
 msgid "Maulidur Rasul"
 msgstr "วันเมาลิดนบี"
+
+#. %s (observed, estimated).
+#, c-format
+msgid "%s (diperhatikan, anggaran"
+msgstr "%ชดเชย%s, โดยประมาณ"
```

Comment:
```suggestion
msgid "%s (diperhatikan, anggaran)"
msgstr "ชดเชย%s, โดยประมาณ"
```
Maybe `msgstr "ชดเชย%s (โดยประมาณ)"`, for consistency with other Thai estimated names translations (with brackets).
@PPsyrius, we need your expertise here.

Reply:
`ชดเชย%s (โดยประมาณ)` would be the correct translation here

Reply:
👍